roaring/roaring.go

```diff
@@ -4066,7 +4343,25 @@ func (op *op) size() int {
 	if op.typ == opTypeAdd || op.typ == opTypeRemove {
 		return 1 + 8 + 4
 	}
-	return 1 + 8 + 4 + len(op.values)*8
+	if op.typ == opTypeAddBatch || op.typ == opTypeRemoveBatch {
+		return 1 + 8 + 4 + len(op.values)*8
+	}
+	// else it's presumably roaring?
```

Comment:
seems like it'd be better to explicitly check this, and have another case for unknown op type... or just check it under roaringParanoia and panic since we don't return an error here

Reply:
Good catch.